enterprise/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to be used across the enterprise djangoapp
+"""
+try:
+    from openedx.core.djangoapps.api_admin.utils import course_discovery_api_client
+except ImportError:
+    course_discovery_api_client = None
+
+class NotConnectedToOpenEdX(Exception):
+    """
+    Exception that is raised when we try to import from
+    the core openedx repository, but it isn't available
+    """
+    pass
+
+
+def get_catalog_api_client(user):
+    """
+    Retrieve an authenticated API client that can be used to access the course
+    discovery API; raise an exception to be caught at a higher level if the
+    package doesn't have OpenEdX resources available.
+    """
+    if course_discovery_api_client is None:
+        raise NotConnectedToOpenEdX(
+            'To get a catalog API client, this package must be installed in an OpenEdX environment.'
```

Comment:
I thought that we only mark error messages for i18n if there's a good chance they'll be seen by users (otherwise it wastes translator time). In this case, is this error likely to be seen by a user using the django admin, or just a one-time error that would be seen by devops when installing this for the first time?